src/main/fc/fc_msp.c

```diff
@@ -142,13 +142,13 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXSONAR, "SONAR", 22 },
     { BOXSERVO1, "SERVO1", 23 },
     { BOXSERVO2, "SERVO2", 24 },
-    { BOXSERVO3, "SERVO3", 25 },
     { BOXBLACKBOX, "BLACKBOX", 26 },
     { BOXFAILSAFE, "FAILSAFE", 27 },
     { BOXAIRMODE, "AIR MODE", 28 },
-    { BOX3DDISABLESWITCH, "DISABLE 3D SWITCH", 29},
-    { BOXFPVANGLEMIX, "FPV ANGLE MIX", 30},
+    { BOX3DPOSSWITCH, "POSTIVE ONLY 3D", 29},
+    { BOX3DNEGSWITCH, "NEGITIVE ONLY 3D", 30},
```

Comment:
For me it is far easier to code up then messing with the mixer, it was a pretty simple change.  Also it is universal solution regardless of what radio someone else.  As well as the RC resolution you mentioned that I didn't think about.

Flipping over a crashed quad is a good use case, but I enjoyed flying 3d (for the first time) by switch as well. 
https://www.youtube.com/watch?v=BZ0wrvzBMwc